tests/helpers/txparser/txtypes.py

```diff
@@ -0,0 +1,204 @@
+from io import BytesIO
+from sys import version_info
+from typing import NewType, Optional, cast
+from dataclasses import dataclass
+assert version_info.major >= 3, "Python 3 required!"
+if version_info.minor >= 8:
+    from typing import Literal
+elif version_info.minor <= 6:   # TypedDict & Literal not yet standard in 3.6
+    from typing_extensions import Literal
+
+# Types of the transaction fields, used to check fields lengths
+u8 = NewType("u8", int)  # 1-byte int
+u16 = NewType("u16", int)  # 2-byte int
+u32 = NewType("u32", int)  # 4-byte int
+u64 = NewType("u64", int)  # 8-byte int
+i64 = NewType("i64", int)  # 8-byte int, signed
+varint = NewType("varint", int)     # 1-9 bytes
+byte = NewType("byte", type(bytes(1)))          # 1 byte
+bytes2 = NewType("bytes2", type(bytes(2)))      # 2 bytes
+bytes4 = NewType("bytes4", type(bytes(4)))      # 4 bytes
+bytes8 = NewType("bytes8", type(bytes(8)))      # 8 bytes
+bytes16 = NewType("bytes16", type(bytes(16)))   # 16 bytes
+bytes32 = NewType("bytes32", type(bytes(32)))   # 32 bytes
+bytes64 = NewType("bytes64", type(bytes(64)))   # 64 bytes
```

Comment:
This doesn't do any type checks apart from checking if the value is a `bytes`. As suggested by @YBadiss on Slack, you can use [a small util from Bluestar](https://github.com/LedgerHQ/python-bluestar/blob/9d891ed78f2a4904a26584eaf9c473ca3684c966/bluestar/hazmat/types/lib.py#L6) that performs runtime checks on the size of `bytes` fields, which is what we should do for tests. You can find some familiar examples [here](https://github.com/LedgerHQ/python-bluestar/blob/master/bluestar/hazmat/types/crypto.py).

If it looks like a lot of work, you can just do:
```py
bytes64 = NewType("bytes64", bytes)   # 64 bytes
```
or just use `bytes` directly, as we're not achieving much with a separate `bytes64` type.

Reply:
Agreed. The `field()` function in Bluestar would be helpful here
https://github.com/LedgerHQ/python-bluestar/blob/master/bluestar/hazmat/types/lib.py#L6

Reply:
Half done:
- Marked the new types as `bytes` rather than as `type(bytes(<size>))`
- Didn't use the `field()` function from Weblue. This is left up to framework maintainers and marked as "TODO" in the test README.